include/deal.II/fe/fe_values.h

```diff
@@ -292,6 +330,24 @@ namespace FEValuesViews
                               std::vector<typename ProductType<value_type,typename InputVector::value_type>::type> &values) const;
 
     /**
+     * Same as above, but using a of vector local degree-of-freedom values.
+     *
```

Comment:
Can you please extend the documentation here to say what you need ``InputVector`` to have? Why do you not use an arrayview or something? Also add that ``dof_values`` is ``n_dofs_per_cell`` long and ``dof_values[i]`` is the value of the local DoF i.